Platform/UTMExtensions.swift

```diff
@@ -97,6 +97,8 @@ extension View {
 extension UTType {
     // SwiftUI BUG: exportedAs: "com.utmapp.utm" doesn't work
     static let UTM = UTType(exportedAs: "utm")
+    
+    static let appleLog = UTType(filenameExtension: "log")!
```

Comment:
Is this needed when there is `UTType.log`?

Reply:
`UTType.log` has a UTI of `public.log`. Console log files have a UTI of `com.apple.log`.

https://developer.apple.com/documentation/uniformtypeidentifiers/uttype/3551527-log

```
% mdls -name kMDItemContentType Library/Containers/com.utmapp.UTM/Data/Documents/Ubuntu\ 20.04\ arm\ 2.utm/debug.log 
kMDItemContentType = "com.apple.log"
```

If you don't specify a 'bottom of the tree' type, NSSavePanel ignores the `allowedContentTypes` and allows all types.

Reply:
TIL